.cirrus.yml

```diff
@@ -89,6 +73,64 @@ task:
   env:
     << : *CIRRUS_EPHEMERAL_WORKER_TEMPLATE_ENV
 
+task:
+  name: "Win64 [unit tests, no functional tests] [msvc]"
+  windows_container:
+    cpu: 8
+    memory: 7424Mi
+    image: cirrusci/windowsservercore:visualstudio2019
+  << : *CREDITS_TEMPLATE
```

Comment:
This looks expensive, especially with 8 CPUs and without `BASE_TEMPLATE` to skip

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/21551#issuecomment-913962298).